### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by maiquanghiep**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             BabylonLabs, Ltd.
+
+Licensed Work:        finality-provider
```
Comment (vitsalis) [inline LICENSE:13]: This is not the licensed work.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             BabylonLabs, Ltd.
```
Comment (vitsalis) [inline LICENSE:11]:
```suggestion
Licensor:             Babylon Labs Ltd.
```

Same across the board

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Licensed Work:        finality-provider
+                      The Licensed Work is (c) 2023 BabylonLabs, Ltd.
+
+Additional Use Grant: None.
```
Comment (vitsalis) [inline LICENSE:16]: We typically add an additional use grant, please make sure that the license is compatible with the one here: https://github.com/babylonlabs-io/babylon/blob/main/LICENSE#L16

**Review by vitsalis — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (maiquanghiep) [reply LICENSE:13]: Should I remove this?

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             BabylonLabs, Ltd.
+
+Licensed Work:        finality-provider
+                      The Licensed Work is (c) 2023 BabylonLabs, Ltd.
```
[the remark this answers is not in the document]
Comment (maiquanghiep) [reply LICENSE:14]: Will update unless we need to delete the `Licensed Work:`

File: LICENSE (hunk 3 above)
Comment (maiquanghiep) [reply LICENSE:16]: Added

File: LICENSE (hunk 1 above)
Comment (vitsalis) [reply LICENSE]: It should include it, please follow exactly the same format as with other repositories and modify only the parameters. The licensed work should be `Babylon GitHub Workflows`

**Review by vitsalis — COMMENTED** [review]
Are the contents of this repository expected to be executed by external teams or are they only intended to be executed by us? Depending on the answer, we might or might not need an additional use grant.

**Comment by maiquanghiep** [comment]
The workflows are mainly tailored to our needs.
However, when someone forks our repo that has the `.github` directory and activate the actions (fork repo has the actions disabled by default), it's likely that it would use workflows created in this repo
Example of such case: https://github.com/maiquanghiep/babylon/actions. I usually fork the repo to test workflows to keep babylonlabs's actions tab clean

**Pull request merged**